### DISCUSSION TITLE
Reason for Name Change?

### DISCUSSION THREAD

**Discussion by CYLITM** [mattwright324/youtube-metadata#151] [General]
I went on the site today and noticed that the website had been renamed from "YouTube Metadata" to "MW Metadata". Is it possible this was to not make it sound like the site is affiliated with YouTube itself?

**Comment by mattwright324** [comment]
Hi @CYLITM 

Yes, as part of a YouTube audit ToS violation I had to change the name, remove the export feature, add "Developed with YouTube" in the footer as well as properly link privacy/terms/contact in the footer as well.
